app/jobs/recalculate_loan_job.rb

```diff
@@ -0,0 +1,14 @@
+class RecalculateLoanJob < ActiveJob::Base
```

Comment:
Can we please call this RecalculateLoanHealthJob, and elsewhere in the code add the 'health' piece? There could be other things about Loans that we are calculating so I want it to be clear this is just about health.